mods/ra/maps/evacuation/evacuation.lua

```diff
@@ -0,0 +1,365 @@
+DeathThreshold =
+{
+	Easy = 200,
+	Normal = 100,
+}
+
+TanyaType = "e7"
+TanyaStance = "AttackAnything"
+if Map.Difficulty ~= "Easy" then
+	TanyaType = "e7.noautotarget"
+	TanyaStance = "HoldFire"
+end
+
+RepairTriggerThreshold =
+{
+	Easy = 50,
+	Normal = 75
+}
+
+Sams = { Sam1, Sam2, Sam3, Sam4 }
+TownUnits =
+{
+	Einstein, Engineer,
+	TownUnit01, TownUnit02, TownUnit03, TownUnit04, TownUnit05, TownUnit06, TownUnit07,
+	TownUnit08, TownUnit09, TownUnit10, TownUnit11, TownUnit12, TownUnit13, TownUnit14
+}
+
+ParabombDelay = DateTime.Seconds(30) 
+ParatroopersDelay = DateTime.Minutes(5)
+Paratroopers =
+{
+	{
+		-- `proxy` cannot be assigned in global scope since a player is needed to create the actor
+		proxy = nil,
+		entry = BadgerEntryPoint1.Location,
+		drop  = BadgerDropPoint1.Location,
+	},
+	{
+		proxy = nil,
+		entry = BadgerEntryPoint1.Location + CVec.New(3, 0),
+		drop  = BadgerDropPoint2.Location,
+	},
+	{
+		proxy = nil,
+		entry = BadgerEntryPoint1.Location + CVec.New(6, 0),
+		drop  = BadgerDropPoint3.Location,
+	},
```

Comment:
I'm not sure what our convention is here. I never add a comma behind the last element of an array. That is up to you though, I won't really complain about it (much). I just wondered as this is even inconsistently done in this single script.